airbyte-integrations/bases/base-normalization/unit_tests/test_transform_catalog.py

```diff
@@ -0,0 +1,156 @@
+"""
+MIT License
+
+Copyright (c) 2020 Airbyte
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+"""
+
+import json
+import os
+import unittest
+
+from normalization import TransformConfig
+from normalization.transform_config.transform import DestinationType
+
+
+class TestTransformCatalog(unittest.TestCase):
```

Comment:
Shouldn't this be `TestTransformConfig` instead? (filename is wrong too?)

Reply:
A potential good (integration?) test that could be made is to run `dbt debug` in the DBT project root directory (in the docker image?) once the profile has been generated.

It will also check if credentials are properly setup and all deps are good to run DBT commands

Reply:
yup. thanks. fixed name.

Reply:
i'm sure we'll have to move the output profile around, but we can figure that out when try to hook up everything together. but agreed that would be a good integration test to write then.